### DISCUSSION TITLE
PLZ Help

### DISCUSSION THREAD

**Discussion by Crepo920** [Godiesc/firefox-gx#197] [General]
![Screenshot_1](https://github.com/user-attachments/assets/76d9fd0c-c245-4895-8a61-994747bda306)

where can i find this logo . i want to set this as my firefox gx logo. if you know where can i finds it will help a lottt. and DaMMM bro This FireFox gx new Virsoin is so cool . Keep it going . Advance Thnx

**Comment by Godiesc** [comment]
hi,
you have all the icons in `chrome/icons` folder, that logo is called [foxmenu.svg](https://github.com/Godiesc/firefox-gx/blob/main/chrome/icons/foxmenu.svg).
